### ISSUE TITLE
Must turn back on vehicle to unlock

### ISSUE THREAD

**Issue by hobart666** [micovery/ArmA3_Wasteland.Altis#94]
After today's update we must turn our backs to the vehicle to get the unlock option to show.

**Comment by codexGW** [comment]
Yeah something is up with the actions on the parking and storage addons as well. I think the addaction needs to use cursor target or the distance set needs to be extended. If I find a solution I'll post it here.

**Comment by hobart666** [comment]
You're the best G-tek thank you for replying. I also have an odd issue with extDB2 I've noticed when I use vactions. If a vehicle is purchased and unlocked upon destruction it doesn't get deleted from the tables if the vehicle stays locked and is destroyed it's deleted from the tables immediately. Any thoughts?

**Comment by hobart666** [comment]
I take it this is what Im looking for? `va_outside_target = {
  ARGVX3(0,_player,objNull);
  ARGVX3(1,_distance,0);
  if (!isPlayer _player) exitWith {};` and change the 0 to 1 for one meter? gonna test this and report back. **Edit** that makes it so one doesn't have to be right up on it at least. should _target or _target_vehicle be replaced with cursorTarget possibly?

**Comment by codexGW** [comment]
Here's the fix. I got it from the A3W forums.
https://github.com/A3Armory/A3A_A3Wasteland_Sock.Altis/commit/137684a9e9951d1dc377aeccd5a8b4ad30429c3a

Here's the fix for the extDB problem I think. Not sure though, we don't use extDB.
http://forums.a3wasteland.com/index.php?topic=943.msg14910#msg14910

**Comment by hobart666** [comment]
Thank you sir I'll give em a shot

**Edit** that first is great as you already know, the second one I've already implemented and thank you.
I know you stated you guys don't use extdb, but do you know what file tells any saving instance the state of a purchased vehicles damage if one is wrecked?
